lib/resdata/tests/rd_smspec_node.cpp

```diff
@@ -273,6 +273,61 @@ void test_cmp_region() {
     test_assert_true(region_node1_2.cmp(region_node2_1) < 0);
 }
 
+void test_history_vectors() {
+    const int dims[3] = {10, 10, 10};
+
+    // valid historical vectors
+    test_assert_true(rd::smspec_node(0, "FOPTH", "UNIT", 0).is_historical());
```

Comment:
Is there something similar to `pytest.mark.parametrize` that can be done here to list each category of vector names to test?